app/src/main/java/fr/neamar/kiss/forwarder/InterfaceTweaks.java

```diff
@@ -119,10 +119,16 @@ private void applyRoundedCorners(MainActivity mainActivity) {
                 mainActivity.findViewById(R.id.searchEditLayout).setBackgroundResource(R.drawable.rounded_search_bar);
             } else {
                 // Before API21, you can't access values from current theme using ?attr/
-                // So we made two different drawables (#931).
+                // So we made four different drawables (#931).
                 if (getSearchBackgroundColor() == Color.WHITE) {
                     mainActivity.findViewById(R.id.externalFavoriteBar).setBackgroundResource(R.drawable.rounded_search_bar_pre21_light);
                     mainActivity.findViewById(R.id.searchEditLayout).setBackgroundResource(R.drawable.rounded_search_bar_pre21_light);
+                } else if (getSearchBackgroundColor() == Color.parseColor("#88FFFFFF")) {
```

Comment:
Can we use the color from colors.xml?

Reply:
done but I'm not sure I've have did it the right way.